### PR TITLE
chore: add Autonolas (OLAS) to validated-tokens

### PR DESCRIPTION
**Pull request by DavidMinarsch**
# Validate [OLAS](https://solscan.io/token/Ez3nzG9ofodYCvEmw73XhQ87LWNYVRM2s7diB5tBZPyM)

## Attestations (Please provide links):
- Tweet from your Twitter Account attesting the Mint address, tagging [@JupiterExchange](https://twitter.com/JupiterExchange) and showing community support: https://twitter.com/autonolas/status/1740295489537667434
- Coingecko/ CMC URL (If available): https://www.coingecko.com/en/coins/autonolas

## Validation (Please check off boxes):
- [x] The metadata provided in the PR matches what is on-chain (Mandatory)
- [x] Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
- [x] Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)  

## Acknowledgement (Please check off boxes)
- [x] My change matches the format in the file (no spaces between fields).
- [ ] My token is already live and trading on Jupiter.
- [x] !!! I read the README section on Community-Driven Validation and understand this PR will be only be reviewed when there is community support on Twitter.
- [x] Please make sure your pull request title has your token name. If it just says "Main", or "Validate", it will automatically be closed. PRs containing broken attestation or solscan links will also be closed.

### CODE REVIEW

File: validated-tokens.csv [hunk 1]
```diff
@@ -1055,3 +1055,5 @@ PONKE,PONKE,5z3EqYQo9HiCEs3R84RCDMu2n7anpDMxRhdK8PSWmrRC,9,https://i.imgur.com/q
 CATWIFHAT,CIF,G3vWvAaXPHCnncnyAbq6yBRXqfRtEV3h7vExzasZeT6g,6,https://i.postimg.cc/hGBgdxxj/Cif200x200.png,true
 AI,AI,99ouK5YUK3JPGCPX9joNtHsMU7NPpU7w91JN4kdQ97po,9,https://gateway.irys.xyz/HnDwG54RZvJhTD2PX-mmrD2jwC9MIdxKjW2hL3M8jQc,true
 catwifhat,$CWIF,7atgF8KQo4wJrD5ATGX7t1V2zVvykPJbFfNeVf1icFv1,2,https://postimg.cc/hJ5rMDvq,true
+Autonolas (Wormhole),$OLAS,Ez3nzG9ofodYCvEmw73XhQ87LWNYVRM2s7diB5tBZPyM,
+8,https://assets.coingecko.com/coins/images/31099/standard/OLAS-token.png,true
```
Comment (oaksprout) [inline validated-tokens.csv:1059]: Can we use our own hosted image?
```suggestion
8,https://olas.network/images/olas-token-logo.svg,true
```
If they don't support svg I can provide a png.

Comment (DavidMinarsch) [reply validated-tokens.csv:1059]: png please :)

Comment (oaksprout) [reply validated-tokens.csv:1059]:
```suggestion
8,https://olas.network/images/olas-token-logo.png,true
```

**Comment by kupermind** [comment]
[x] Mint address: Ez3nzG9ofodYCvEmw73XhQ87LWNYVRM2s7diB5tBZPyM
[x] Symbol: OLAS
[x] Token Decimals: 8
[x] Token icon: https://assets.coingecko.com/coins/images/31099/standard/OLAS-token.png
[x] On-chain metadata matches
[x] No duplication on Jupiter's strict list
[x] Community tweet: https://x.com/kupermind/status/1740302083432333649?s=20

**Comment by oaksprout** [comment]
- [x]  Mint address: Ez3nzG9ofodYCvEmw73XhQ87LWNYVRM2s7diB5tBZPyM
- [x]  Symbol: OLAS
- [x]  Token Decimals: 8
- [x]  Token icon: https://olas.network/images/olas-token-logo.png
- [x]  On-chain metadata matches
- [x]  No duplication on Jupiter's strict list

**Review by benlvb — APPROVED** [review]

**Comment by Timoon21** [comment]
Hey @DavidMinarsch I don't see any logoURI on the token metadata https://solscan.io/token/Ez3nzG9ofodYCvEmw73XhQ87LWNYVRM2s7diB5tBZPyM#metadata
Can you update your token metadata to match the logoURI in the PR?

**Comment by DavidMinarsch** [comment]
> Hey @DavidMinarsch I don't see any logoURI on the token metadata https://solscan.io/token/Ez3nzG9ofodYCvEmw73XhQ87LWNYVRM2s7diB5tBZPyM#metadata Can you update your token metadata to match the logoURI in the PR?

This is a bridged token. We have no control over the permissionless creation process

**Comment by DavidMinarsch** [comment]
Hey @Timoon21 any thoughts on the response above? Would be great to get this merged.

**Comment by benlvb** [comment]
Screenshot 2024-01-23 at 19 06 44

There seems to be a discrepancy on the token name and ticker.

**Comment by DavidMinarsch** [comment]
> Screenshot 2024-01-23 at 19 06 44
> 
> There seems to be a discrepancy on the token name and ticker.

By this logic every token needs to be rejected. The scanner displays the name and the ticker. E.g. take the first entry in the list (https://github.com/jup-ag/token-list/blob/main/validated-tokens.csv) and compare it against solscan https://solscan.io/token/CbNYA9n3927uXUukee2Hf4tm3xxkffJPPZvGazc2EAH1

**Comment by DavidMinarsch** [comment]
As to why the "(Wormhole)" now has been dropped by the scanner, I have no idea. This was not the case when this PR was opened. Would be great to get this merged one way or the other.

**Pull request closed without merging**